app.js

```diff
@@ -1,35 +1,48 @@
 
-var pElOne = document.getElementById('answerOne');
-var pElTwo = document.getElementById('answerTwo');
-var pElThree = document.getElementById('answerThree');
-
 var els = [
-  pElOne,
-  pElTwo,
-  pElThree,
+  document.getElementById('answerOne'),
+  document.getElementById('answerTwo'),
+  document.getElementById('answerThree'),
+  document.getElementById('answerFour'),
+  document.getElementById('answerFive'),
 ];
 
 var questions = [
   "What is your name?",
```

Comment:
You ask the user what their name is, then compare it to your own name. There's a logic error here.